src/briefcase/integrations/android_sdk.py

```diff
@@ -713,12 +716,12 @@ def start_emulator(self, avd):
 
             # Step 1: Wait for the device to appear so we can get an
             # ADB instance for the new device.
-            print()
-            print('Waiting for emulator to start...', flush=True, end='')
+            self.command.input.print()
+            self.command.input.print('Waiting for emulator to start...', flush=True, end='')
```

Comment:
This is another progress bar... (this time one with undetermined length); it's not an input condition.

Reply:
Created a wait bar context manager in Console. Implemented each place this sort of thing is happening.